canvas_cli/tests.py

```diff
@@ -82,23 +82,28 @@ def write_plugin(
     with chdir(integration_tests_plugins_dir):
         cli_runner.invoke(app, "init", input=plugin_name)
 
-    protocol_code = """
+    handler_code = """
+from canvas_sdk.effects import Effect
 from canvas_sdk.events import EventType
-from canvas_sdk.protocols import BaseProtocol
+from canvas_sdk.handlers import BaseHandler
 from logger import log
 
-class Protocol(BaseProtocol):
-    RESPONDS_TO = EventType.Name(EventType.ASSESS_COMMAND__CONDITION_SELECTED)
-    NARRATIVE_STRING = "I was inserted from my plugin's protocol."
 
-    def compute(self):
-        log.info(self.NARRATIVE_STRING)
+class NewOfficeVisitNoteHandler(BaseHandler):
+    \"\"\"Handler for integration tests.\"\"\"
```

Comment:
Why all these escapes?

I see you've done it everywhere in the tests, any reason for it?

Reply:
this is inside a verbatim string, without the escapes it would end it right? an alternative to the escapes would be to use `'''` instead (or use a `'''` verbatim string and `"""` inside the the verbatim string)